pwiz_tools/Skyline/Skyline.cs

```diff
@@ -3303,7 +3303,11 @@ public void ShowPeptideSettingsUI(PeptideSettingsUI.TABS? tab)
 
         private void ShowPeptideSettingsUI(IWin32Window parent, PeptideSettingsUI.TABS? tab)
         {
-            using (PeptideSettingsUI ps = new PeptideSettingsUI(this, _libraryManager) { TabControlSel = tab })
+            using (PeptideSettingsUI ps = new PeptideSettingsUI(this, _libraryManager) { TabControlSel = tab ?? (PeptideSettingsUI.TABS)

```

Comment:
Let's put this back inside PeptideSettingsUI in the constructor.

Reply:
Done.